website/static/js/onboarder.js

```diff
@@ -596,6 +597,10 @@ function OBUploaderViewModel(params) {
     };
     self.dropzone = new Dropzone(self.selector, dropzoneOpts);
 
+    if($osf.isIE()){
+        $('h4:contains(1. Drop file (or click below))').replaceWith('<h4>1. Click below to select file</h4>');
```

Comment:
I considered that first, but @lyndsysimon suggested this method. I'll use an ID if you think that's best.